packages/react/src/components/controls/StartAudio.tsx

```diff
@@ -44,6 +46,20 @@ export function useStartAudio({ room, props }: UseStartAudioProps) {
     [props, className, canPlayAudio, handleStartAudioPlayback, roomEnsured],
   );
 
+  log.warn(`canPlayAudio: ${canPlayAudio}`);
+
+  const dummyAudio = React.useRef<HTMLAudioElement>();
+
+  React.useEffect(() => {
+    if (canPlayAudio && !dummyAudio.current) {
+      dummyAudio.current = document.createElement('audio');
+      dummyAudio.current.autoplay = true;
+      //@ts-ignore
+      dummyAudio.current.playsInline = true;
```

Comment:
Is there a reason why `playsInline` is set? It seems to be just a "video" tag attribute 🤔.

Reply:
ah, yeah, that makes sense, no reason. will remove it